### PR TITLE
Adjust scrollable-tab-bar style

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
Style has changed

### CODE REVIEW

File: src/components/Activity/ScrollableTabBar.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
+const React = require('react');
```
Comment (MechanicKim) [inline src/components/Activity/ScrollableTabBar.js:1]: Is this file a part of react native package?

Comment (ifndefdeadmau5) [reply src/components/Activity/ScrollableTabBar.js:1]: Correct

File: src/components/Activity/Button.ios.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  View,
+} = ReactNative;
+
+const Button = (props) => {
+  return <TouchableOpacity {...props}>
+    {props.children}
+  </TouchableOpacity>;
+};
+
+module.exports = Button;
```
Comment (heartjigi) [inline src/components/Activity/Button.ios.js:14]: add line plz

**Comment by MechanicKim** [comment]
Good! I heard that about this change.
Thank you.
LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/116/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**